Neos.ESCR.AssetUsage/Classes/Dto/NodeAddress.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Neos.ContentRepository package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\ESCR\AssetUsage\Dto;
+
+use Neos\ContentRepository\Core\DimensionSpace\DimensionSpacePoint;
+use Neos\ContentRepository\Core\SharedModel\Node\NodeAggregateId;
+use Neos\ContentRepository\Core\SharedModel\Workspace\ContentStreamId;
+use Neos\ContentRepository\Core\SharedModel\Workspace\WorkspaceName;
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * Is used to address a node within the AssetUsage package
+ *
+ * @internal
+ */
+#[Flow\Proxy(false)]
+final class NodeAddress
```

Comment:
Hm, can't we use the NodeAddress here? I more closely need to check the code, but I think having such a copied object is kinda dangerous (with the same name)

Reply:
Yes, there is a NodeAddress within the FrontendRouting https://github.com/dlubitz/neos-development-collection/blob/a43ac99132245143a20fae4eed50ae9b8de58b7f/Neos.Neos/Classes/FrontendRouting/NodeAddress.php#L42

But this is just similar and can't get reused. Since this `NodeAddress` is for internal use only, the naming would not bother me, but I'm open for any suggestions!?